x/poolmanager/client/cli/cli_test.go

```diff
@@ -74,9 +74,9 @@ func TestNewSwapExactAmountOutCmd(t *testing.T) {
 	desc, _ := cli.NewSwapExactAmountOutCmd()
 	tcs := map[string]osmocli.TxCliTestCase[*types.MsgSwapExactAmountOut]{
 		"swap exact amount out": {
-			Cmd: "10stake 20 --swap-route-pool-ids=1 --swap-route-denoms=node0token --from=" + testAddresses[0].String(),
+			Cmd: "10stake 20 --swap-route-pool-ids=1 --swap-route-denoms=node0token",
 			ExpectedMsg: &types.MsgSwapExactAmountOut{
-				Sender:           testAddresses[0].String(),
+				Sender:           "",
```

Comment:
why are we removing Sender from `SwapExactAmountOutCmd`? Shouldn;t `SwapExactAmountOutCmd` still have the sender?

Reply:
yes! I just removed it from the wrong place (search and replace)